cfgov/v1/atomic_elements/atoms.py

```diff
@@ -154,3 +155,33 @@ def clean(self, data):
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
         else:
             return data
+
+class AnchorLink(blocks.StructBlock):
+    link_id = blocks.CharBlock(required=False)
```

Comment:
> output a field in the admin that prints the relative URL with the anchor link appended, for easy copy and paste for a content editor.

How does the StructBlock allow you to achieve this? We should chat about what you mean.